src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <TimeZoneDataVersion>2020a</TimeZoneDataVersion>
+      <TimeZoneSource>https://data.iana.org/time-zones/tzdb</TimeZoneSource>
+      <InputFolder>$(MSBuildProjectDirectory)/input</InputFolder>
+      <OutputFolder>$(MSBuildProjectDirectory)/zoneinfo</OutputFolder>
```

Comment:
Please use the IntermediateOutputPath for this instead so that these won't show up as changes in the repo.